lib/client.js

```diff
@@ -401,6 +403,7 @@ Client.prototype._connectCb = function (callback) {
   }
   this.connecting = true;
   const self = this;
+  this.log('info', util.format("Connecting to cluster using '%s' version %s", description, version));
```

Comment:
Will log something like:

> Connecting to cluster using 'DataStax Node.js Driver for Apache Cassandra' version 3.4.1


I surrounded the package description in quotes to remove ambiguity, as without them it's possible that one may interpret the version as the Apache Cassandra version.